### PR TITLE
7 localize german

### PR DESCRIPTION
**Pull request by appfrosch**
# What it Does
* adding German translations (see issue #7)

# How I Tested
* Ran the application
* Set the Sim device to German
* Checked all the currently available screens

# Notes
* it will be an ongoing effort to check for new strings to translate during development
* I will check in regularily

# Screencast
https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/91844195/625edc0b-b348-4bbf-9158-0c9171bb8796

### CODE REVIEW

File: Basic-Car-Maintenance.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -623,7 +624,7 @@
 				MACOSX_DEPLOYMENT_TARGET = 13.4;
 				MARKETING_VERSION = 1.0;
 				OTHER_LDFLAGS = "-ObjC";
-				PRODUCT_BUNDLE_IDENTIFIER = "com.icyappstudio.Basic-Car-Maintenance";
+				PRODUCT_BUNDLE_IDENTIFIER = "ch.appfros.Basic-Car-Maintenance";
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance.xcodeproj/project.pbxproj]: Don't change the bundle ID please! Go back and fix all the instances of this

**Review by mikaelacaron — CHANGES_REQUESTED** [review]
Please fix the bundle ID change and then click the review button

Re-review button

File: Basic-Car-Maintenance.xcodeproj/project.pbxproj (hunk 1 above)
Comment (appfrosch) [reply Basic-Car-Maintenance.xcodeproj/project.pbxproj]: Sorry, that's quite obviously not a good idea... As per our small conversation on Mastadon, I changed it cause it gave me an error for macOS–but committing it in the PR doesn't make sense anyway…

**Review by mikaelacaron — APPROVED** [review]
Great work!

**Pull request merged**